File: multi-gammon-util/src/main/java/org/akoshterek/backgammon/train/AgentSettings.java
package org.akoshterek.backgammon.train;

import org.akoshterek.backgammon.agent.inputrepresentation.InputRepresentation;
import org.akoshterek.backgammon.agent.inputrepresentation.PointCodec;

/**
 * @author Alex
 *         date 22.09.2015.
 */
public class AgentSettings {
    public InputRepresentation representation;
    public PointCodec pointCodec;
    public int hiddenNeuronCount;
}

File: multi-gammon-util/src/main/java/org/akoshterek/backgammon/train/AgentSettings.scala
package org.akoshterek.backgammon.train

import org.akoshterek.backgammon.agent.inputrepresentation.InputRepresentation
import org.encog.engine.network.activation.ActivationFunction

class AgentSettings (val representation: InputRepresentation,
                     val hiddenNeuronCount: Int,
                     val agentName: String,
                     val activationFunction: ActivationFunction){
}